docs/packages/markdown/parseMarkdown.js

```diff
@@ -6,7 +6,7 @@ const prism = require('./prism');
 const headerRegExp = /---[\r\n]([\s\S]*)[\r\n]---/;
 const titleRegExp = /# (.*)[\r\n]/;
 const descriptionRegExp = /<p class="description">(.*?)<\/p>/s;
-const headerKeyValueRegExp = /(.*?): (.*)/g;
+const headerKeyValueRegExp = /(.*?):[\r\n]?\s+(\[[^\]]+\]|.*)/g;
```

Comment:
Adding many authors doesn't work because `.*` matches any character except newlines, so in the value below only the key is matched:

```
authors: 
  ['foo', 'bar']
```

We could use `(.|\n)*`, however in the first pass it would match everything.

The new regex instead only matches newlines if the value begins with `[` and ends with `]`. If not, it will only match the characters.